### DISCUSSION TITLE
A few questions on dataset preparation

### DISCUSSION THREAD

**Discussion by johannphilippe** [acids-ircam/RAVE#61] [Q&A]
Hello, and many thanks for this exciting project. 

I understand that the bigger a dataset is, the better it is. But, I'd like to know if there are some things in particular to take care of when preparing the dataset before training. For example : 
* Is there a recommanded size for samples ? Long samples ? Short samples ? A mix of both ? 
* I guess fadein/out at the ends of samples (and in general all "non relevant" information) are not so good for training ?
* In general, anything you use to take care of before starting training would be useful.

Thanks !

**Comment by jreus** [comment]
As far as I know RAVE will chop up your audio files into chunks of size `N_SIGNAL` during training. The default value for `N_SIGNAL` is 65536, so about 1.36 seconds. My guess is that this should be your minimal audio file size. (@caillonantoine  please correct if I'm wrong!)

Naturally you want to get rid of any non relevent information in the data. The model will try to fit whatever you give to it ... noise and all.
